native/common/jp_methoddispatch.cpp

```diff
@@ -45,46 +46,57 @@ bool JPMethodDispatch::findOverload(JPJavaFrame& frame, JPMethodMatch &bestMatch
 	JP_TRACE("Checking overload", m_Name);
 	JP_TRACE("Got overloads to check", m_Overloads.size());
 	JPMethodList ambiguous;
-	JPMethodMatch match(frame, arg);
+	JPMethodMatch match = bestMatch;
```

Comment:
can't this be a reference (avoid a copy)?

Reply:
Unfortunately no.  We need to two copies here.  One is the used to hold the best match we have found and one is used to test the next match (which destroys the state of the match).   We are doing a copy here because we happened to have run the same command out one layer.   There was no reason to rerun the command when we could just copy the results.

Reply:
I see. Thanks!